Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM debian:latest@sha256:f2150eba68619015058b26d50e47f9fba81213d1cb81633be7928c
 LABEL maintainer="Liblouis Maintainers <liblouis-liblouisxml@freelists.org>"
 
 # Fetch build dependencies
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get -y dist-upgrade && apt-get install -y \
```

Comment:
We should not do a `dist-upgrade`. We specify the Debian version using the `FROM` keyword